src/main/java/ch/njol/skript/expressions/ExprInventoryChest.java

```diff
@@ -38,32 +38,31 @@
 @Name("Custom Chest Inventory")
 @Description("Returns a chest inventory with the given amount of rows and the name. Use the <a href=effects.html#EffOpenInventory>open inventory</a>effect to open it.")
 @Examples({"open chest inventory with 1 rows named \"test\"",
-           "set {_inv} to chest inventory with 1 row"})
+           "set {_inventory} to chest inventory with 1 row"})
 @Since("2.2-dev34")
-public class ExprNewInventory extends SimpleExpression<Inventory> {
+public class ExprInventoryChest extends SimpleExpression<Inventory> {
 
     static {
-        Skript.registerExpression(ExprNewInventory.class, Inventory.class, ExpressionType.COMBINED, "chest inventory (named|with name) %string% [with %-number% row[s]]"
-                                                                                                , "chest inventory with %number% row[s] [(named|with name) %-string%]");
+        Skript.registerExpression(ExprInventoryChest.class, Inventory.class, ExpressionType.COMBINED, "[a [new]] chest [inventory] (named|with name) %string% [with %-number% row[s]]", "chest [inventory] with %number% row[s] [(named|with name) %-string%]");
```

Comment:
You added `[a [new]]` to the first pattern but what about the second one? Also, don't make `[inventory]` optional.

Reply:
Why can't the `inventory` be optional? I tested three different cases like you said and I could not get them to conflict in the slightest.

Reply:
Even if it doesn't conflict it isn't consistent with Skript inventory types' names, I guess this can be an exception since making `inventory` optional would make it works with scripts using skQuery for inventory opening.

Reply:
Also it should be ``[a] [new]``, "a" isn't required at anywhere like this 😛